processing/forward_handler.go

```diff
@@ -185,6 +186,9 @@ func (fh *ForwardHandler) Consume(e *types.Entry) error {
 		if err != nil {
 			return err
 		}
+		if GlobalContextCollector != nil && e.EventType == "alert" {
```

Comment:
Another candidate for constant `EvenTypeAlert`.

Reply:
Done